src/stream/src/executor/aggregation/agg_group.rs

```diff
@@ -237,16 +238,41 @@ impl<S: StateStore, Strtg: Strategy> AggGroup<S, Strtg> {
         self.group_key.as_ref()
     }
 
+    pub fn group_key_row(&self) -> OwnedRow {
+        self.group_key
+            .as_ref()
+            .map(GroupKey::table_row)
+            .cloned()
+            .unwrap_or_default()
+    }
+
     fn prev_row_count(&self) -> usize {
         match &self.prev_outputs {
             Some(states) => states[self.row_count_index]
                 .as_ref()
-                .map(|x| *x.as_int64() as usize)
+                .map(|x| {
+                    TryInto::try_into(*x.as_int64()).expect("row count should be non-negative")
+                })
                 .unwrap_or(0),
             None => 0,
         }
     }
 
+    /// Get current row count of this group.
+    fn curr_row_count(&self) -> StreamExecutorResult<usize> {
+        let row_count_state = must_match!(
+            self.states[self.row_count_index],
+            AggState::Value(ref state) => state
+        );
+        let row_count = row_count_state
+            .get_output()
+            .expect("row count should never output NULL")
+            .into_int64();
+        Ok(row_count
+            .try_into()
+            .map_err(|_| anyhow!("row count should be non-negative, but got {}", row_count))?)
```

Comment:
I guess it's okay to panic here directly.

Reply:
By returning an error we can print more context info at the caller frame

Reply:
Why panic can’t? What’s the difference? T👀

Reply:
Returning an error to `AggGroup::apply_chunk` can enable us to log the current chunk processing?

Reply:
But looks like logging the chunk won't help too much🤣 I don't know🥵

Reply:
Changed.